content/recipes/nestia.md

```diff
@@ -0,0 +1,243 @@
+
+```bash
+# COMPILE THROUGH TTYPESCRIPT
+npx ttsc
+
+# RUN TS-NODE WITH TTYPESCRIPT
+npx ts-node -C ttypescript src/index.ts
+```
+
+##### Manual Setup
+
+If you want to install and configure `nestia` manually, read [Guide Documents - Setup](https://github.com/samchon/nestia/wiki/Setup).
+
+
+
+
+
+#### `@nestia/core`
+
+Superfast validation decorators for NestJS.
+
+  - 15,000x faster request body validation
+  - 10x faster JSON response, even type safe
+  - Do not need DTO class definition, just fine with interface
+
+`@nestia/core` is a transformer library of NestJS, supporting superfast validation decorators, by wrapping [typia](https://github.com/samchon/typia). Comparing validation speed with `class-validator`, `typia` is maximum **15,000x times faster**, and it is even much safer.
+
+Furthermore, `@nestia/core` can use pure interface typed DTO with **only one line**. With `@nestia/core`, you don't need any extra dedication like defining JSON schema (`@nestjs/swagger`), or using class definition with decorator function calls (`class-validator`). Just enjoy the superfast decorators with pure TypeScript type.
+
+```typescript
+import { Controller } from "@nestjs/common";
+import { TypedBody, TypedRoute } from "@nestia/core";
+
+import type { IBbsArticle } from "@bbs-api/structures/IBbsArticle";
+
+@Controller("bbs/articles")
+export class BbsArticlesController {
+    /** 
+     * Store a new content.
+     * 
+     * @param inupt Content to store
+     * @returns Newly archived article
+     */
+    @TypedRoute.Post() // 10x faster and safer JSON.stringify()
+    public async store(
+        @TypedBody() input: IBbsArticle.IStore // superfast validator
+    ): Promise<IBbsArticle>; 
+        // do not need DTO class definition, 
+        // just fine with interface
+}
+```
+
+##### TypedBody
+
+`TypedBody()` is a decorator function of `application/json` typed request body.
+
+Also, it supports superfast validation pipe, which is maximum **15,000x times faster** than `nest.Body()` function using `class-validator`.
+
+##### TypedRoute
+
+`TypedRoute` is a set of decorator functions for `application/json` typed response body.
+
+Also, it supports safe and fast JSON stringify function pipe, which is maximum 10x times faster than native `JSON.stringify()` function. Furthermore, it is **type safe** through validation.
+
+  - `TypedRoute.Get()`
+  - `TypedRoute.Post()`
+  - `TypedRoute.Put()`
+  - `TypedRoute.Patch()`
+  - `TypedRoute.Delete()`
+
+##### Comment Tags
+
+You can enhance DTO type validation by writing comment tags.
+
+If you want to know more about it, read [Guide Documents of `typia`](https://github.com/samchon/typia/wiki/Runtime-Validators#comment-tags).
+
+```typescript
+export interface IBbsArticle {
+    /**
+     * @format uuid
+     */
+    id: string;
+
+    writer: IBbsArticle.IWriter;
+
+    /**
+     * @minItems 1
+     */
+    contents: IBbsArticle.IContent[];
+}
```

Comment:
Are you JSDoc comments necessary, or is that for dev knowledge?

Reply:
Not neccessary, but just for additional type checking if user wanta